www/src/components/dodont/index.tsx

```diff
@@ -5,32 +5,73 @@ import styles from './index.module.scss';
 
 interface PropTypes {
     children: React.ReactNode;
-    type: 'do' | 'dont';
+    type?: 'do' | 'dont';
+    coloredHeading?: boolean;
+    isTable?: boolean;
```

Comment:
The table parameter was added to allow for direct comparisons of copy juxtapositioned. This also allows for far less redundancy that would otherwise overcrowd the page with visual do/don't callouts that could detract from the overall goal.